daemon/api_systems.go

```diff
@@ -209,3 +213,8 @@ func postSystemActionDo(c *Command, systemLabel string, req *systemActionRequest
 	}
 	return SyncResponse(nil)
 }
+
+func postSystemActionInstall(c *Command, systemLabel string, req *systemActionRequest) Response {
```

Comment:
we should check that systemLabel is not empty, no?

Reply:
you resolved this, but there's no new check. Are you expecting do the check in the called function?

Reply:
Meh, that was a mistake, I initially added this but testing at this level is a bit annoying as the mux framework will not call this when systemLabel is empty so I figured I will do it in the new `DeviceManager.InstallStep()` call (as it's here needed anyway too).